### PR TITLE
Update yarn config

### PR DESCRIPTION
**Pull request by dblodorn**
# Yarn config updates
TLDR Spent a bunch of time messing around with yarn 2/3 but just ended up modifying the yarn config a little and adding a clean command - and slipped an edit to the changesets config defaulting the access property to public [https://github.com/changesets/changesets/issues/503](https://github.com/changesets/changesets/issues/503)

## ☠️ Upgrading to yarn 3 notes--- 🥴

Ok so the new version of Yarn (yarn >3.X) is kinda hellish and has introduced numerous bugs tracing into dependencies of dependencies. Bottom line it seems like a lot of work to get everything working correctly and the older version of yarn just works... seemingly for most everyone using it, and that's better for a template which is meant to be an easy starting point.

I'm going to revert back to a stable setup...

just a pile of errors like this - that were not an issue with older version of yarn:

Screen Shot 2022-09-22 at 9 25 56 AM

Initial motivation to upgrade was the excitment around using `@yarnpkg/plugin-workspace-tools` - this resulted in needed to upgrade the version of yarn i was using to a version that supported yarn plugins: [https://yarnpkg.com/features/plugins](https://yarnpkg.com/features/plugins)

## Some links collected from this rabbit hole 🕳:
[https://yarnpkg.com/getting-started/migration](https://yarnpkg.com/getting-started/migration)
[https://github.com/yarnpkg/berry/issues/454](https://github.com/yarnpkg/berry/issues/454)
[https://yarnpkg.com/api/modules/plugin_workspace_tools.html](https://yarnpkg.com/api/modules/plugin_workspace_tools.html)
[https://dev.to/arcanis/yarn-3-0-performances-esbuild-better-patches-e07](https://dev.to/arcanis/yarn-3-0-performances-esbuild-better-patches-e07)
[https://github.com/parcel-bundler/parcel/issues/7697](https://github.com/parcel-bundler/parcel/issues/7697)
[https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored](https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored)
[https://www.digitalocean.com/community/tutorials/how-to-install-and-use-the-yarn-package-manager-for-node-js](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-the-yarn-package-manager-for-node-js)
[https://semaphoreci.com/blog/javascript-monorepos-yarn-workspaces](https://semaphoreci.com/blog/javascript-monorepos-yarn-workspaces)
[https://twitter.com/arcanis/status/1301151739375161344?lang=en](https://twitter.com/arcanis/status/1301151739375161344?lang=en)
[https://stackoverflow.com/questions/71747609/how-to-specify-packagemanager-in-package-json](https://stackoverflow.com/questions/71747609/how-to-specify-packagemanager-in-package-json)

### CODE REVIEW

**Comment by dblodorn** [comment]
I've retreated from the new versions of yarn and am staying with yarn classic.
